ILMerge.Tests/TestFiles.cs

```diff
@@ -1,8 +1,29 @@
-﻿namespace ILMerging.Tests
+﻿using NUnit.Framework;
+using System.IO;
+
+namespace ILMerging.Tests
 {
     internal static class TestFiles
     {
-        public const string TestSnk = "test.snk";
-        public const string TestPfx = "test.pfx";
+        private static string FromCurrentDir(string fileName)
+        {
+            return Path.Combine(TestContext.CurrentContext.TestDirectory, fileName);
+        }
+
+        public static string TestSnk
+        {
+            get
+            {
+                return FromCurrentDir("test.snk");
+            }
+        }
```

Comment:
That syntax is C# 6, so anyone with VS2015 or newer can use it. If @mike-barnett does not intend to allow the very latest C# it would be a good idea to set the language level in the csprojs.